app/src/main/res/drawable/chapter_white_bg_with_bright_green_border.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+  android:shape="rectangle">
+  <stroke android:color="@color/color_def_bright_green" android:width="4dp"/>
+  <solid android:color="@color/color_def_white"/>
```

Comment:
PLease file and issue mentioning that this new design needs Dark-mode mock too.

Reply:
done